table.cpp

```diff
@@ -109,6 +102,11 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   } else if (event == "Extension"){
     extension = true;
     inquiry = bad_status = override = version_conflict = cancelled = false;
+      if (reason != ""){
+      hoverText = "class=\"hoverable-cell\" data-hover-text=\"" + userName + " received a " + std::to_string(daysExtended) + " day extension due to " + reason + " on " + gID + "\" ";
+      } else {
+      hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension without specified reason on "+gID+"\" ";
```

Comment:
line missing indentation

Reply:
Updated in [f9402f2](https://github.com/Submitty/RainbowGrades/pull/77/commits/f9402f2b1b2950980ccf7fa3093f3155c4256321)